Dockerfile

```diff
@@ -9,16 +9,15 @@ RUN yum update -y && \
     bzip2 \
     epel-release \
     gcc-c++ \
-    openssl-devel
+    openssl-devel \
+    wget \
+    unzip
```

Comment:
After specifying the `unzip` package, be sure to clean up the local yum cache.
E.g.:
```
    unzip && \
    yum clean all
```

Also, the `epel-release` package isn't valid for Amazon Linux 2 and so can be removed from the list.